fix(respect): secrets masking for encoded occurrences in har-output (#2998)
Redocly/redocly-cli: .changeset/mask-encoded-secrets.md

---
'@redocly/respect-core': patch
---

> Fixed secrets masking to cover encoded occurrences of a secret in har-output.

Reviewer: ```suggestion
Fixed an issue where secrets masking did not cover encoded secrets in `har-output`.
```
